code/modules/projectiles/projectile/bullets.dm

```diff
@@ -232,6 +232,12 @@
 	maim_type = DROPLIMB_BLUNT
 	anti_materiel_potential = 2
 
+/obj/item/projectile/rifle/kumar_super
+	damage = 35
+	armor_penetration = 20
+	weaken = 2
```

Comment:
Get rid of this as a stun is a bit too powerful. Otherwise, looks fine, I guess?

Reply:
As you wish